plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcConnectorFactory.java

```diff
@@ -36,13 +37,13 @@ public class JdbcConnectorFactory
         implements ConnectorFactory
 {
     private final String name;
-    private final Module module;
```

Comment:
Oh crap, this looked so benign!
thanks for fixing this.
how do we make sure there are no other cases like this one, in other connectors?